src/main/kotlin/no/nav/syfo/motebehov/MotebehovService.kt

```diff
@@ -27,10 +27,16 @@ class MotebehovService @Inject constructor(
 ) {
     @Transactional
     fun behandleUbehandledeMotebehov(arbeidstakerFnr: Fodselsnummer, veilederIdent: String) {
-        val antallOppdateringer = motebehovDAO.oppdaterUbehandledeMotebehovTilBehandlet(aktorregisterConsumer.getAktorIdForFodselsnummer(arbeidstakerFnr), veilederIdent)
-        if (antallOppdateringer > 0) {
+        val arbeidstakerAktorId = aktorregisterConsumer.getAktorIdForFodselsnummer(arbeidstakerFnr)
+        val motebehovUbehandletList = motebehovDAO.hentUbehandledeMotebehov(arbeidstakerAktorId)
+        if (motebehovUbehandletList.isNotEmpty()) {
             val behandlendeEnhet = behandlendeEnhetConsumer.getBehandlendeEnhet(arbeidstakerFnr.value, null).enhetId
-            oversikthendelseService.sendOversikthendelseBehandlet(arbeidstakerFnr, behandlendeEnhet)
+            motebehovUbehandletList.forEach { pMotebehov ->
```

Comment:
hva er p en i pMotebehov?

Reply:
P står for persistency.
https://github.com/navikt/syfomotebehov/blob/be81c5c6ffc1a1932d774598eb7dfde3eb0255a2/src/main/kotlin/no/nav/syfo/motebehov/database/PMotebehov.kt